Update & improve tutorial window

Moneyl opened the pull request:
- Fix scroll bar not showing up till window is resized/scrolled.
- Update keybinds and information.
- Add "quick controls" before detailed explanation for quick reference.
- Add section headers with larger font.
- Add paragraph on death & respawning

To make the paragraph formatting look decent I had to make a few lines of code for defining the strings very long. Tried using string linebreaks but they don't seem to work with formatted messages.

Before: (orange A was from me testing formatting, can ignore)
![Content Client_Mx9u3byqHV](https://user-images.githubusercontent.com/8206401/73629843-4e9d8580-4622-11ea-8b8b-2a0f485db307.png)

After:
![Content Client_wC9LRkrhAj](https://user-images.githubusercontent.com/8206401/73631968-6e37ac80-4628-11ea-8695-ab0c232ca112.png)

RemieRichards commented:
If you're listing the controls like this now, why repeat it in the paragraph still?
The list of "Action: Keybind" is much better than the paragraph.

PJB3005 commented on Content.Client/UserInterface/GameHud.cs:
```diff
@@ -267,6 +267,8 @@ private void ButtonTutorialOnOnToggled()
                 _tutorialWindow.OpenCentered();
                 _buttonTutorial.Pressed = true;
+                //Fixes scrollbar not showing up until window is resized/scrolled.
+                _tutorialWindow.ForceRunLayoutUpdate();
```
This really should not be the correct way to fix this issue. The documentation for `ForceRunLayoutUpdate` specifically says that needing to use it most certainly signifies a bug in the UI system, and this is one such place I feel that holds.

Moneyl replied:
Will see if I can find the issue and make a separate PR with a fix.

Moneyl commented:
Yeah I agree that most of the paragraph is unnecessary now. I removed most of it except for the sentences on death, respawning, and combat mode since they might not be immediately obvious, and stuck them in a "gameplay" section. 

![Content Client_MdgEe4kbeT](https://user-images.githubusercontent.com/8206401/73702797-e8654100-46bb-11ea-9768-08631084996e.png)

Removed the use of `ForceRunLayoutUpdate`. Will try to figure out why the scrollbar isn't showing up until resize and make a separate PR. Will probably be an engine change anyway.

PJB3005 commented on Content.Client/UserInterface/TutorialWindow.cs:
```diff
@@ -1,51 +1,90 @@
+";
 
-X switches hands. Z uses the item in your hand. Q drops items. T focuses chat. C opens your inventory.
+        private const string GameplayContents = @"Some notes on gameplay. To talk in OOC, prefix your chat message with \[ or /ooc. Death is currently show as a black circle around the player. You can respawn via the respawn button in the sandbox menu. Instead of intents, we have ""combat mode"". Check controls above for it's keybind. You can't attack anybody with it off, so no more hitting yours ...
```
Typo on `it's keybind`

Moneyl replied:
Fixed. Will add a combat mode melee explanation a bit later
